### PR TITLE
MTF : Answer options are not displayed completely

### PR DESCRIPTION
**Pull request by JagadishPujari**

### CODE REVIEW

**Comment by manojchandrashekar** [comment]
Changes are OK. Please show me a demo of how it works before merge happens. @JagadishPujari

File: renderer/js/mtftemplate.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 var QS_MTFTemplate = {}; 
 
 QS_MTFTemplate.htmlLayout = "<div class='mtf-layout'>\
-    <header class='mtf-header'>\
-      <div class='mtf-question-text'><%= question.data.question.text %>\</div>\
+    <header class='mtf-header' id='scrollbar'>\
```
Comment (manojchandrashekar) [inline renderer/js/mtftemplate.js]: Why is id needed?

File: renderer/styles/style.css [hunk 2]
```diff
@@ -8,15 +9,18 @@
 header.mtf-header {
     width: 100%;
     background-color: #404041;
-    height: auto;
+    height: 21vh;
+    margin-bottom: 4%;
+    overflow-y: scroll;
+    /* overflow-x: hidden; */
```
Comment (manojchandrashekar) [inline renderer/styles/style.css]: Commented code?

**Comment by manojchandrashekar** [comment]
@vinukumar-vs Please review.

File: editor/controllers/horizontal_controller.js [hunk 3]
```diff
@@ -171,9 +170,17 @@ angular.module('mtfApp', [])
       return (formValid && opSel) ? true : false;
     }
     $scope.deletePair = function(id) {
+      $scope.indexCount--;
       $scope.mtfFormData.option.optionsLHS.splice(id, 1);
       $scope.mtfFormData.option.optionsRHS.splice(id, 1);
-      //}
```
Comment (manojchandrashekar) [inline editor/controllers/horizontal_controller.js]: Please remove commented code!

**Comment by manojchandrashekar** [comment]
@vinukumar-vs FYA

File: editor/controllers/horizontal_controller.js [hunk 4]
```diff
@@ -6,7 +6,7 @@
 angular.module('mtfApp', [])
   .controller('mtfQuestionFormController', ['$scope', '$rootScope', function($scope, $rootScope) {
     $scope.formVaild = false;
-    $scope.indexCount = 4;
+    $scope.indexCount = 3;
```
Comment (vinukumar-vs) [inline editor/controllers/horizontal_controller.js]: What is this indexCount?

File: editor/controllers/horizontal_controller.js [hunk 5]
```diff
@@ -171,9 +170,9 @@ angular.module('mtfApp', [])
       return (formValid && opSel) ? true : false;
     }
     $scope.deletePair = function(id) {
+      $scope.indexCount--;
```
Comment (vinukumar-vs) [inline editor/controllers/horizontal_controller.js]: Why are you using indexCount here?

**Pull request merged**